README.md

```diff
@@ -99,6 +101,16 @@ Get source code, run installer and follow on screen instructions:
 git clone https://github.com/AdnanHodzic/auto-cpufreq.git
 cd auto-cpufreq && sudo ./auto-cpufreq-installer
 ```
+### Update using auto-cpufreq-installer
```

Comment:
Let's remove changes made here from line 104 to 113, as they are just repeating how to install auto-cpufreq-installer and update is explained as part of separate section below.

Reply:
shall i add the following line with some edits to line 296?
"The feature is available in version >= *1.9.9*. If your current version is below the same, Get the source code and run the installer"
?

Reply:
No, I would put it to `The feature is available in version >= 1.9.8. If your current version is below the same, Get the source code and run the installer`.

Because this functionality will be available immediately on >= 1.9.8 for whoever's using auto-cpufreq-installer, we don't need to wait for a release to have it available.

Reply:
ok

Reply:
i get it what you are saying, just rephrasing:
you want to add it immediately but want the package to update when a new package is released, because
if this is the case then I'll only need to change the version_value
else
if you want to update on each new commit made, I need to find another alternative.

Reply:
No, this is fine for now.